LAMBERT_AIMPOINT_GUIDANCE.s

```diff
@@ -65,7 +65,7 @@
 
 # DISPLAYS USED IN P-31LM **
 #
-# V06N33	DISPLAY SOTRED TIG (IN HRS. MINS. SECS.)
+# V06N33	DISPLAY STORED TIG (IN HRS. MINS. SECS.)
```

Comment:
`SOTRED` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/0651.jpg)